data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/handlers/v1/beacon/PostAttestation.java

```diff
@@ -87,7 +72,10 @@ public PostAttestation(
                         .getAttestationSchema()
                         .getJsonTypeDefinition()))
             .response(SC_OK, "Attestations are stored in pool and broadcast on appropriate subnet")
-            .response(SC_BAD_REQUEST, "Errors with one or more attestations", BAD_REQUEST_RESPONSE)
+            .response(
+                SC_BAD_REQUEST,
+                "Errors with one or more attestations",
+                ErrorListBadRequest.getFailuresTypeDefinition())
```

Comment:
nit: should probably call this `getJsonTypeDefinition`?

Reply:
But there are two different type definitions on that class. Can't call them both `getJsonTypeDefinition`